### PR TITLE
tooling(storybook): add more storybook stories for existing components

### PR DESCRIPTION
**Pull request by billyvg**
Adds storybook stories for some of our existing components

#nochanges

### CODE REVIEW

**Comment by getsentry-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Changes to build requirements</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: webpack.config.js [hunk 1]
```diff
@@ -177,10 +178,11 @@ var config = {
         : path.join(__dirname, 'src/sentry/integration-docs/_platforms.json')
     },
     modules: [path.join(__dirname, staticPrefix), 'node_modules'],
-    extensions: ['*', '.jsx', '.js', '.json']
+    extensions: ['.less', '.jsx', '.js', '.json']
   },
   output: {
     path: distPath,
+    publicPath: '/_static/sentry/dist/',
```
Comment (benvinegar) [inline webpack.config.js]: Are we using code splitting or anything here? I learned later this doesn't work and fixed it [in this PR](https://github.com/getsentry/sentry/pull/5623/commits/50b5ee383371741f64b835682c8e8faa63996b3d).

**Review by benvinegar — CHANGES_REQUESTED** [review]
Please see my `publicPath` comment, otherwise lgtm.

File: webpack.config.js (hunk 1 above)
Comment (billyvg) [inline webpack.config.js]: Nope, I think this was added in the old styleguide when we wanted to codesplit, so seems good to remove.

**Review by benvinegar — APPROVED** [review]

**Pull request merged**